suite-native/module-accounts-import/src/components/AccountImportOverview.tsx

```diff
@@ -54,28 +30,14 @@ export const AccountImportOverview = ({
                     </Text>
                 </Box>
                 <Text variant="label" color="gray1000">
-                    ≈ {formatNetworkAmount(accountInfo.availableBalance, 'btc', true)}
+                    ≈ {formatNetworkAmount(accountInfo.availableBalance, currencySymbol, true)}
                 </Text>
             </Box>
             <Box marginBottom="large">
                 <InputWrapper>
                     <Input value={assetName} onChange={onAssetNameChange} label="" />
```

Comment:
isInvalid={accountName.lengt !== 0 && <check max length>}
- apply on button

Reply:
Don't do that now, I am already started to prepare forms package and this should be handled as form, so after forms will be merged we will implement it here.

Reply:
Nice, I suppose it will be [react-hook-form](https://react-hook-form.com/), @Nodonisko?

Reply:
Yea

Reply:
Why not have some basic functionality of this when it only involves changing couple of lines? Can be refactored to  hook form anyway.

Reply:
I suppose we will use different validation approach later, e.g. with Yup validator or sth like that `Yup.string().nullable().min(......` so any kind of temporary validation functions are not important for now. And there will definitely be some issue later to walk through the code and add missing forms, default values and validators...

Reply:
Okay. It would be nice to have it but fair enough :D 

Can you please create issue for future refactor just so we know it's not finished yet?

Reply:
It would require quite a lot of work to do it properly with only useState and that is probably waste right now.